lib/linter-julia.coffee

```diff
@@ -23,9 +23,9 @@ doSomeMagic = (data,textEditor) ->
   lines = data.split("\n")
   for line in lines
     try
-      splittedline = line.split(":")
-      numbers = splittedline[1].split(" ")
-      if numbers[1] in ignore
+      splittedline = line.replace(filePath+":","").split(":")
+      numbers = splittedline[0].split(" ")
+      if numbers.length < 2 || numbers[1] in ignore
```

Comment:
I was hitting a problem where `numbers` was an array with only one element.  To tell the truth, I'm not sure why this was happening.  At the time I made the change, I thought that it was because I was shrinking the string before doing the split, but, actually, I'm not sure if that makes sense.  Anyways, thanks for merging, and glad to hear there is another solution in the works.